js/main.js

```diff
@@ -1,3 +1,4 @@
-import {createPhotos} from './mockPhotos.js';
-createPhotos();
+import './mockPhotos.js';
+import './renderingPicture.js';
+
```

Comment:
И тут же в этом файле main.js теперь мы выполняем логику "основного потока" нашего приложения
```suggestion
// Получаем данные
const photos = createPhotos();

// И их отрисовываем
renderPhotos(photos);
```